File: SofiForce.Models/Models/EntityModels/ComplainTypeDetailModel.cs
using System;

namespace Models
{
    public class ComplainTypeDetailModel
    {
        public int ComplainTypeDetailId { get; set; }
        public int? ComplainTypeId { get; set; }
        public string ComplainTypeDetailCode { get; set; }
        public string ComplainTypeDetailNameAr { get; set; }
        public string ComplainTypeDetailNameEn { get; set; }
        public bool? IsActive { get; set; }
        public bool? CanEdit { get; set; }
        public bool? CanDelete { get; set; }
        public int? DisplayOrder { get; set; }
        public string Color { get; set; }
        public string Icon { get; set; }
        public int? CBy { get; set; }
        public DateTime? CDate { get; set; }
        public int? EBy { get; set; }
        public DateTime? EDate { get; set; }
    }
}

File: SofiForce.Web/ClientApp/src/app/core/Models/EntityModels/ComplainTypeDetailModel.ts
export interface ComplainTypeDetailModel {
    complainTypeDetailId: number;
    complainTypeId: number | null;
    complainTypeDetailCode: string;
    complainTypeDetailNameAr: string;
    complainTypeDetailNameEn: string;
    isActive: boolean | null;
    canEdit: boolean | null;
    canDelete: boolean | null;
    displayOrder: number | null;
    color: string;
    icon: string;
    cBy: number | null;
    cDate: Date | null;
    eBy: number | null;
    eDate: Date | null;
}